site/_includes/partials/script.js

```diff
@@ -39,11 +39,14 @@
     '{{ analytics.version }}'
   );
   try {
-    ga(
-      'set',
-      '{{ analytics.dimensions.NAVIGATION_TYPE }}',
-      performance.getEntriesByType('navigation')[0].type.replace(/_/g, '-')
-    );
+    // For the document speculation rules origin trial
+    // overrwrite the navigation type
+    const navEntry = performance.getEntriesByType('navigation')[0];
```

Comment:
Do you need to handle UAs which don't support the performance timeline API (by feature detecting it)?

Reply:
I actually don't think we need to here as in a try/catch already.